### ISSUE TITLE
[suggestion] Please add to FAQ "Is it Foss" item

### ISSUE THREAD

**Issue by DubbaThony** [midjourney/docs#44]
Since Source and model aren't FOSS, I would expect many people wasting a lot of time trying to figure out if it is or not (especially it's hosted @ gitbook, some GH repos _do exist_ etc).

I suggest updating FAQ page to contain this information.

**Comment by Tiltedtable** [comment]
Worth a shot

**Comment by Arthavruksha** [comment]
https://github.com/midjourney/docs/pull/50 Solved

**Comment by lucasgonze** [comment]
Yes, I am here figuring that out, and it's not a good use of time.
